crates/napi/src/bindgen_runtime/js_values/bigint.rs

```diff
@@ -125,8 +124,7 @@ impl BigInt {
     if len == 1 {
       (self.sign_bit, self.words[0] as u128, false)
     } else {
-      let u128_words: [u64; 2] = [self.words[0], self.words[1]];
-      let val = unsafe { ptr::read(u128_words.as_ptr() as *const u128) };
+      let val = self.words[0] as u128 + ((self.words[1] as u128) << 64);
```

Comment:
This change does not change the asm output as well.
[before](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=83476c6d68be053daf09f066f9e0ac30), [after](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=2a8e7b113071b62bfb47ef9b30367c33)